content/en/docs/concepts/windows/intro.md

```diff
@@ -371,8 +371,6 @@ troubleshooting ideas prior to creating a ticket.
 
 The kubeadm tool helps you to deploy a Kubernetes cluster, providing the control
 plane to manage the cluster it, and nodes to run your workloads.
-[Adding Windows nodes](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/)
```

Comment:
Did we intend to delete this, @tengqm?

Reply:
https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/ is no longer there (we redirect to the most similar page that does exist).

Reply:
> Did we intend to delete this, @tengqm?

Yes. It was suggested by the Windows contents "owner".

Reply:
Thank you for the clarification, folks! Much appreciated :)